api/envoy/extensions/filters/http/grpc_field_extraction/v3/config.proto

```diff
@@ -22,6 +22,11 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // request message no matter if it is unary or streaming and it write the result
 // to the destination, for which currently only the static dynamic metadata `envoy.filters.http.grpc_field_extraction` is supported.
 //
+// # Assumptions
+// 1. this filter is only applicable for gRPC with Protobuf as payload.
+// 2. for bi-di and client-side gRPC streaming, the client-side body emission should not depend on receiving server initial metadata.
```

Comment:
The wording here is a little unclear.  Suggest replacing "client-side body emission" with "initial message from the client".

Reply:
Done.